### ISSUE TITLE
human texture not work on android

### ISSUE THREAD

**Issue by HyundongHwang** [natmlx/videokit#72]
# introduction
- The human texture of the VideoKitCameraManager does not work on Android devices.
- It operates normally in the Windows Unity player.
- In particular, on Android devices, if the capability of the VideoKitCameraManager is set to Everything, both the human texture and camera texture become empty.
- I am using the VideoKit AI license.
<img width="500" alt="image" src="https://github.com/natmlx/videokit/assets/5696570/cff96a28-8f2d-4bfa-9958-c4bc0ca44113">

# test environment
- host : windows 11 
- target : android 13
  - samsung galaxy s20 ultra

# windows unity player
<img width="400" alt="image" src="https://github.com/natmlx/videokit/assets/5696570/9f2e12b2-4b19-4cfb-9d11-a77b32045eac">

# android app
<img width="400" alt="image" src="https://github.com/natmlx/videokit/assets/5696570/6a3d8ad3-46b3-48ce-895e-644fa29c31e7">

**Comment by olokobayusuf** [comment]
Hey @HyundongHwang , sorry for the late response on this. Can you share the runtime logs from your Android device?

**Comment by HyundongHwang** [comment]
@olokobayusuf 
Thank you for your kind reply.
However, I did not solve this problem, but avoided it by bypassing the implementation.
After rendering in CamDevice, I did some modifications in Unity, rendered the screen, stored it in RenderTexture, and used it as input in NatML's Predicators, so there was no problem.
Hopefully, other developers will probably need more modifications rather than using human textures directly, so I would like to suggest that it might be better to add guidance in the document like this.
So it became difficult to prepare the log, so I think we can close the issue.
